api/src/test/java/uk/gov/dhsc/htbhf/claimant/entitlement/PaymentCycleVoucherEntitlementTest.java

```diff
@@ -148,4 +150,14 @@ void shouldReturnLastVoucherEntitlement() {
         assertThat(lastVoucherEntitlement).isEqualTo(minusOneWeek);
     }
 
+    @Test
+    void shouldReturnCorrectBackdatedVouchersValue() {
+        VoucherEntitlement voucherEntitlement = aValidVoucherEntitlement();
+        int backdatedVouchers = 10;
+
+        PaymentCycleVoucherEntitlement result = new PaymentCycleVoucherEntitlement(asList(voucherEntitlement), backdatedVouchers);
+
+        assertThat(result.getBackdatedVouchersValueInPence()).isEqualTo(backdatedVouchers * VOUCHER_VALUE_IN_PENCE);
```

Comment:
This should be an explicit value rather than a calculation, it makes it harder to figure out if the test is correct or not.

Reply:
Do you mean 1:
```
int expected = backdatedVouchers * VOUCHER_VALUE_IN_PENCE;
assertThat(result.getBackdatedVouchersValueInPence()).isEqualTo(expected);
```
.. or 2:
```
assertThat(result.getBackdatedVouchersValueInPence()).isEqualTo(3100);
```
?

Reply:
I mean option 2, it's perfectly clear then what you're asserting.

Reply:
I would argue that the meaning of the number is then less clear. It's just a magic number.

Reply:
We're compromising by including a comment with the calculation

Reply:
I don't agree with you there, I always feel it's better to be completely specific over what we are asserting and to not have your expected value be a calculation. Option 2 doesn't contain (or need) a comment either so I'm not sure what you mean there.

Reply:
Updated